package.json

```diff
@@ -31,6 +31,7 @@
     "jszip": "^3.10.1",
     "lucide-vue-next": "^0.321.0",
     "qr-code-styling": "^1.6.0-rc.1",
+    "qr-code-styling-utf8": "^1.9.4",
```

Comment:
I don't understand where this library is from. I can't see it on npm. I don't think it's a good idea to use unverified libraries.

Reply:
it would be ideal to get the an appropriate utf8 fix up on the main qr-code-styling library, and then update its version here rather than having 2 separate qr-code-styling library dependencies.

Reply:
Reading the https://github.com/kozakdenys/qr-code-styling/issues/62, it seems that you can just encode the string before passing it to the qr-code-styling library to create the QR. Could you give it a try and see if that suffices?